profile/README.md

```diff
@@ -0,0 +1,70 @@
+<picture style="display: block; max-width: 100%">
+  <source media="(prefers-color-scheme: dark)" srcset="../artwork/transparency-dev-dark.png">
+  <source media="(prefers-color-scheme: light)" srcset="../artwork/transparency-dev-light.png">
+  <img alt="Transparency.dev logo" src="../artwork/transparency-dev-light.png">
+</picture>
+<br/>
+
+Welcome to transparency.dev, a community dedicated to promoting transparency technologies across various ecosystems and pioneering its integration into new application domains.
+
+Tamper-evident logs are the heart of transparency technology and are relied on by [Certificate Transparency](https://certificate.transparency.dev/), [Sigstore (Supply Chain Security)](https://www.sigstore.dev/), and many other ecosystems. Visit [transparency.dev](https://transparency.dev/) to learn more!
+
+## 🔨 Transparency.dev Projects
+
+This repository hosts critical projects for building end-to-end transparency ecosystems, including:
+- [trillian-tessera](https://github.com/transparency-dev/trillian-tessera) - a Go library for building tile-based transparency logs
+- [static-ct](https://github.com/transparency-dev/static-ct) - An implementation of the Static CT API based on Tessera
+- [witness](https://github.com/transparency-dev/witness) - Libraries and binaries for running witnesses for verifiable logs
+- [armored-witness](https://github.com/transparency-dev/armored-witness) - A plug and play witness solution intended to kick-start a cross-ecosystem witness network
+
+## ⚒️ Related Projects
+
+Explore a variety of related transparency projects through the following links:
+
+- [trillian](https://github.com/google/trillian) - A transparent, highly scalable and cryptographically verifiable data store.
+- [C2SP](https://github.com/C2SP/C2SP) - Community Cryptography Specification Project, home to transparency log specifications
+- [sunlight](https://github.com/FiloSottile/sunlight) - A Certificate Transparency log implementation and monitoring API designed for scalability, ease of operation, and reduced cost.
+
+## 🌳 Roadmap
+
+Here's a glimpse of some of the things we're working on during the upcoming months, along with highlights of recently completed initiatives.
+For more details, check out the links or join us on [Slack](https://join.slack.com/t/transparency-dev/shared_invite/zt-27pkqo21d-okUFhur7YZ0rFoJVIOPznQ) to discuss. 
+
+### ✅ Recently Completed
```

Comment:
You could also highlight the Transparency.dev 2024 summit? Maybe https://transparency.dev/summit2024/wednesday.html since it has links to all the presentations, etc?

Reply:
Great idea, done!